java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1063,6 +1083,33 @@ public void lpush_lpop_lrange_type_error(BaseClient client) {
         assertTrue(lrangeException.getCause() instanceof RequestException);
     }
 
+    //     TODO: uncomment once client.lrange has binary version
```

Comment:
You can use non-binary `lrange`, but leave a TODO to replace it there with a binary version.
Test should pass since you're not using real binary strings here.